### ISSUE TITLE
how to change am/pm time for transit isochrone picker

### ISSUE THREAD

**Issue by rmglennon** [mapzen/mobility-explorer#215]
What if we put both AM and PM on the UI? The active one can be highlighted and it gives an easy and more obvious way to switch between them.

![image](https://cloud.githubusercontent.com/assets/4380498/20024647/34d6c630-a2a5-11e6-8ca4-e3090a4baac9.png)

Maybe something like this

![image](https://cloud.githubusercontent.com/assets/4380498/20024826/3597afca-a2a6-11e6-88a3-b8a7a982011f.png)

**Comment by drewda** [comment]
@rmglennon FYI, this date/time picker is a prebuilt component that @meghanhade is importing: http://eonasdan.github.io/bootstrap-datetimepicker/ If it comes with an existing option to make a change like this, it's a piece of cake. If not, it's probably more trouble than it's worth.

**Comment by meghanhade** [comment]
@rmglennon @drewda I totally agree, Rhonda. I tried to change this when I first implemented the time picker, but it was more trouble than it was worth at that time. Probably something that should be improved in the future!
